script_automation/esa_ubuntu_k3_single/pv.template.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+    name: esa-pv
+    namespace: default
+spec:
+    capacity:
+        ### This storage capacity value is not enforced at this layer. ###
+        storage: 10Gi
```

Comment:
Maybe ytou need to add a prereq to deploy the cluster with at least 10GB of free space